csrc/scheduler/mma_utils.cpp

```diff
@@ -691,37 +691,6 @@ void WarpMmaSwizzler::scheduleLdMatrix(TensorView* tv, MmaOptions options) {
     //[2ko, 2mno, 8k, 8mni]                 [2ko, 1mno, 8k, 8mni]
   }
 
-  // ldmatrix loads multiple 8x8 matrices from shared memory to registers in a
```

Comment:
According to the unit test result, this should be handled in the allocation domain instead of the leaf domain